projects/ui/src/lib/components/po-button/po-button-base.component.ts

```diff
@@ -121,16 +111,21 @@ export class PoButtonBaseComponent {
    * Define o estilo do `po-button`.
    *
    * Valore válidos:
-   *  - `default`: estilo padrão do `po-button`.
+   *  - `default`: **Deprecated 16.x.x** Usar `secondary`.
+   *  - `danger`: **Deprecated 16.x.x** Usar `secondary` em conjunto com a propriedade `p-danger`.
+   *  - `link`: **Deprecated 16.x.x** o `po-button` recebe o estilo de um link.
    *  - `primary`: deixa o `po-button` com destaque, deve ser usado para ações primárias.
-   *  - `danger`: deve ser usado para ações que o usuário precisa ter cuidado ao executa-lá.
-   *  - `link`: o `po-button` recebe o estilo de um link.
+   *  - `secondary`: estilo padrão do `po-button`.
+   *  - `tertiary`: o `po-button` é exibido sem cor de fundo, recebendo menos destaque entre as ações.
    *
-   * @default `default`
+   * @default `secondary`
    */
   @Input('p-type') set type(value: string) {
-    this._type = PO_BUTTON_TYPES.includes(value) ? value : PO_BUTTON_TYPE_DEFAULT;
+    this._type = PO_BUTTON_TYPES.includes(value) ? value : PoButtonType[value] ? PoButtonType[value] : PO_BUTTON_TYPE_DEFAULT;
```

Comment:
Se for alterado de link para tertiary, não dá para manter a declaração anterior?
```this._type = PO_BUTTON_TYPES.includes(value) ? value : PO_BUTTON_TYPE_DEFAULT;```